pkg/cmd/server/plugin/plugin.go

```diff
@@ -40,11 +40,17 @@ func NewCommand(f client.Factory) *cobra.Command {
 				RegisterBackupItemAction("velero.io/pv", newPVBackupItemAction).
 				RegisterBackupItemAction("velero.io/pod", newPodBackupItemAction).
 				RegisterBackupItemAction("velero.io/service-account", newServiceAccountBackupItemAction(f)).
-				RegisterBackupItemAction("velero.io/crd-remap-version", newRemapCRDVersionAction(f)).
+				// We disable the crd-remap-version plugin because otherwise CRDs with version v1beta1 won't get backed
+				// up properly.
+				//RegisterBackupItemAction("velero.io/crd-remap-version", newRemapCRDVersionAction(f)).
```

Comment:
FYI: This will be reverted with an upcoming commit.

Reply:
See: https://github.com/c445/velero/pull/2/commits/7505550eab23dcb43bbd50f3387ae18907af28c7